bin/create_hugo_data.py

```diff
@@ -494,13 +494,149 @@ def export_events(anthology, builddir, dryrun):
+                        "first": speaker.name.first if speaker.name.first else "",
+                        "last": speaker.name.last,
+                    }
+                    talk_data["speakers"].append(speaker_dict)
+                # Add video URL if available
+                if "video" in talk.attachments:
+                    talk_data["video_url"] = talk.attachments["video"].url
+                # Add other attachments
+                attachments = []
+                for att_type, attachment in talk.attachments.items():
+                    if att_type != "video":
+                        attachments.append({"type": att_type, "url": attachment.url})
+                if attachments:
+                    talk_data["attachments"] = attachments
+                talks_data.append(talk_data)
+            data["talks"] = talks_data
+
         all_events[event.id] = data
 
     if not dryrun:
         with open(f"{builddir}/data/events.json", "wb") as f:
             f.write(ENCODER.encode(all_events))
 
 
+def export_talks(anthology, builddir, dryrun):
+    # Export individual talk data files
+    all_talks = {}
+    print("Exporting talks...")
+
+    os.makedirs(f"{builddir}/data/talks", exist_ok=True)
+
+    for event in anthology.events.values():
+        if not event.talks:
+            continue
+
+        for idx, talk in enumerate(event.talks, 1):
+            # Generate talk ID from video filename if available, otherwise use default pattern
+            if "video" in talk.attachments and talk.attachments["video"].name:
+                # Extract talk ID from video filename like "2022.acl-keynote.2.mp4"
+                video_name = talk.attachments["video"].name
+                if video_name.endswith(".mp4"):
+                    # Remove .mp4 extension to get the talk ID
+                    talk_id = video_name[:-4]
+                else:
+                    talk_id = video_name
+            else:
+                # Fallback to sequential numbering if no video
+                talk_id = f"{event.id}.talk-{idx}"
+
+            # Create talk data
+            talk_data = {
+                "id": talk_id,
+                "event_id": event.id,
+                "event_title": event.title.as_text() if event.title else f"{event.id}",
+                "title": talk.title.as_text(),
+                "title_html": remove_extra_whitespace(
+                    talk.title.as_html(allow_url=False)
+                ),
+                "speakers": [],
+                "type": talk.type,
+            }
+
+            # Process speakers
+            for speaker in talk.speakers:
+                speaker_data = {
+                    "id": speaker.id if speaker.id else "",
+                    "full_name": speaker.name.as_first_last(),
+                    "first": speaker.name.first if speaker.name.first else "",
+                    "last": speaker.name.last,
+                }
```

Comment:
This should use `person_to_dict()` like everywhere else in the build script; "full_name" will be just "full" then, though, might need to be adapted in the template.